autopush/base.py

```diff
@@ -39,8 +39,8 @@ def write_error(self, code, **kwargs):
             self.log.failure(
                 format=kwargs.get('format', "Exception"),
                 failure=failure.Failure(*kwargs['exc_info']),
-                **self._client_info)
+                client_info=self._client_info)
```

Comment:
This will mess up our Kibana logging, where we do want the client_info dict as additional keys.